web/package.json

```diff
@@ -93,6 +94,7 @@
     "@integreat-app/eslint-config-jest": "^0.6.0",
     "@integreat-app/eslint-config-node": "^0.6.0",
     "@integreat-app/eslint-config-react": "^0.6.1",
+    "@testing-library/react": "^11.0.4",
```

Comment:
In the web project we use enzyme which is able to render components in many different ways. Did you try enzyme first?

Reply:
Yeah, I did. I think the testing library is a better solution and encourages better test. I didn't found a nice way to test the stuff I wanted to test here, so I decided to introduce the testing library in the web project as well. Imo we should switch entirely in the long run.

Reply:
k, should we create a task for that? We can also just use both libraries

Reply:
I think this could go hand in hand with general refactoring and could also be done incrementally? I mean it is not wrong to use enzyme, I just think the testing library encourages better tests. Also, the tests I replaced did not work at all...

Reply:
Do you think we should create a task for this?

Reply:
Probably not worth the effort. I'd say we do this when it starts to bug us. Did you check the enzyme API whether this if really not possible with enzyme?

```
it('should show correct text', () => {
  const wrapper = shallow(<MyComponent />);
  expect(wrapper.text().includes('my text')).toBe(true);
});
```

Reply:
No, did not check this api. Just tried the approach that was used before (searching for components).